tests/cases/fourslash/generatorDeclarationFormatting.ts

```diff
@@ -7,4 +7,4 @@ format.document();
 goTo.marker('1');
 verify.currentLineContentIs("function* g() { }");
 goTo.marker('2');
-verify.currentLineContentIs("var v = function* () { };");
\ No newline at end of file
+verify.currentLineContentIs("var v = function*() { };");
```

Comment:
🚨 Note that treating `SpaceAfterAnonymousFunctionKeyword` differently actually changes the long-standing default for anonymous generators. If that's undesirable, I can probably add a FormattingContext filter, but personally I think this is probably more consistent.